cl/redisapp/leaderfollower/leaderfollower.go

```diff
@@ -44,12 +44,23 @@ type BlockBuilder interface {
 	SetLastCallTimeToZero()
 }
 
+// todo: work with block state through block builder, not directly
```

Comment:
This todo shouldn't affect single node consensus correct?

Reply:
yes, correct